src/node/CarplayNode.ts

```diff
@@ -15,16 +13,28 @@ import {
   DongleDriver,
   DongleConfig,
   DEFAULT_CONFIG,
+  Key,
+  CarPlay,
+  AudioCommand,
 } from '../modules'
 
-export default class CarplayWS extends EventEmitter {
+const USB_WAIT_PERIOD_MS = 500
+const USB_WAIT_RESTART_MS = 3000
+
+export type CarplayMessage =
+  | { type: 'plugged'; message?: undefined }
```

Comment:
Having `message` typed for items that don't need it makes downstream usescases easier, ie you can do 

```
x.onmessage = function ({type, message})
```
without Typescript yelling at you that `message` doesn't exist as a property